types/emoji-regex/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for emoji-regex 7.0
+// Project: https://mths.be/emoji-regex
+// Definitions by: iKBAHT <https://github.com/iKBAHT>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function createRegExp(): RegExp;
+export default createRegExp;
```

Comment:
It should be `export = createRegExp ` https://github.com/mathiasbynens/emoji-regex/blob/master/src/index.js#L1